corehq/apps/userreports/tests/test_pillow.py

```diff
@@ -611,19 +611,20 @@ def _test_process_doc_from_sql_stale(self, pillow=None, num_queries=4):
 
         http://manage.dimagi.com/default.asp?245341
         '''
-
-        pillow = pillow or self.pillow
-        for i in range(3):
-            since = pillow.get_change_feed().get_latest_offsets()
-            form, cases = self._post_case_blocks(i)
-            with self.assertNumQueries(num_queries):
-                pillow.process_changes(since=since, forever=False)
-            rows = self.adapter.get_query_object()
-            self.assertEqual(rows.count(), 1)
-            row = rows[0]
-            self.assertEqual(int(row.parent_property), i)
-            errors = PillowError.objects.filter(doc_id='child-id', pillow=pillow.pillow_id)
-            self.assertEqual(errors.count(), 0)
+        with mock.patch('corehq.pillows.case.KafkaCheckpointEventHandler.should_update_checkpoint') as fake_checkpoint:
```

Comment:
Nit: I think you can do `mock.patch('...', return_value=False)` and then not have to name the variable or set the return value on a second line.